src/collision.h

```diff
@@ -36,37 +36,21 @@ enum CollisionType
 
 struct CollisionInfo
 {
-	enum CollisionType type;
-	v3s16 node_p; // COLLISION_NODE
-	bool bouncy;
-	v3f old_speed;
-	v3f new_speed;
-
-	CollisionInfo():
-		type(COLLISION_NODE),
-		node_p(-32768,-32768,-32768),
-		bouncy(false),
-		old_speed(0,0,0),
-		new_speed(0,0,0)
-	{}
+	CollisionType type = COLLISION_NODE;
+	v3s16 node_p = v3s16(-32768,-32768,-32768); // COLLISION_NODE
+	bool bouncy = false;
+	v3f old_speed = v3f(0.0f, 0.0f, 0.0f);
+	v3f new_speed = v3f(0.0f, 0.0f, 0.0f);
```

Comment:
They're defined to be [`0,0,0` in the constructor without args](http://irrlicht.sourceforge.net/docu/vector3d_8h_source.html#l00026l)